numpy/core/tests/test_multiarray.py

```diff
@@ -1512,7 +1512,7 @@ def test_dumps(self):
 
     def test_pickle(self):
         for proto in range(2, pickle.HIGHEST_PROTOCOL + 1):
-            for dt in [bytes, np.void, unicode]:
+            for dt in [bytes, np.void, str]:
                 zs = self._zeros(10, dt)
                 p = pickle.dumps(zs, protocol=proto)
                 zs2 = pickle.loads(p)
```

Comment:
Below in line 4459 there is another one of these (causes the test failures). Probably I will just fix it up myself and merge later.